test/BinaryDBALEventStoreTest.php

```diff
@@ -61,7 +56,8 @@ public function table_should_contain_binary_uuid_column()
         $uuidColumn = $this->table->getColumn('uuid');
 
         $this->assertEquals(16, $uuidColumn->getLength());
-        $this->assertEquals(Type::getType(Type::BINARY), $uuidColumn->getType());
+
+        $this->assertEquals(Type::getType('binary'), $uuidColumn->getType());
```

Comment:
Constants have moved to `Types`, so you can use `Types::BINARY`

Reply:
Thanks! This felt wrong indeed :)